Makefile

```diff
@@ -3,20 +3,19 @@
 REGISTRY ?= quay.io
 REPOSITORY ?= $(REGISTRY)/kohlstechnology
 
-BUILD_COMMIT := $(shell ./scripts/build/get-build-commit.sh)
-BUILD_TIMESTAMP := $(shell ./scripts/build/get-build-timestamp.sh)
-BUILD_HOSTNAME := $(shell ./scripts/build/get-build-hostname.sh)
+COMMIT=`git rev-parse HEAD`
+BRANCH=`git symbolic-ref --short -q HEAD || echo HEAD`
+DATE=`date -u +%Y%m%d-%H:%M:%S`
```

Comment:
$() could be used instead of backticks - they are considered [legacy syntax](https://unix.stackexchange.com/a/126928)

Reply:
@sbar95 fixed in 8c33c73cbbaf765b26fd30bd0a267caf5e286c7c